src/traceProvider.ts

```diff
@@ -11,58 +11,58 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-import {
-  BatchSpanProcessor,
-  NodeTracerProvider,
-  NoopSpanProcessor,
-} from '@opentelemetry/sdk-trace-node';
-import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
-import { Resource } from '@opentelemetry/resources';
-import { HttpInstrumentation } from '@opentelemetry/instrumentation-http';
-import { GrpcInstrumentation } from '@opentelemetry/instrumentation-grpc';
-import { registerInstrumentations } from '@opentelemetry/instrumentation';
-import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
-import { TraceIdRatioBasedSampler } from '@opentelemetry/sdk-trace-node';
+// import {
+//   BatchSpanProcessor,
+//   NodeTracerProvider,
+//   NoopSpanProcessor,
+// } from '@opentelemetry/sdk-trace-node';
+// import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
+// import { Resource } from '@opentelemetry/resources';
+// import { HttpInstrumentation } from '@opentelemetry/instrumentation-http';
+// import { GrpcInstrumentation } from '@opentelemetry/instrumentation-grpc';
+// import { registerInstrumentations } from '@opentelemetry/instrumentation';
+// import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
+// import { TraceIdRatioBasedSampler } from '@opentelemetry/sdk-trace-node';
```

Comment:
Can these just be removed rather than commented out? Same with the others below.